action.yml

```diff
@@ -225,6 +273,29 @@ runs:
 
         TERRAFORM_RESULT=$?
 
+        if [[ "${{ inputs.pr-comment }}" == "true" ]]; then
+          tfcmt \
+          --config ${{ github.action_path }}/config/summary.yaml \
+          -owner "${{ github.repository_owner }}" \
+          -repo "${{ github.event.repository.name }}" \
+          -var "target:${{ steps.vars.outputs.component_slug }}" \
+          -var "component:${{ inputs.component }}" \
+          -var "componentPath:${{ steps.vars.outputs.component_path }}" \
+          -var "commitSHA:${{ inputs.sha }}" \
+          -var "stack:${{ inputs.stack }}" \
+          -var "job:${{ github.job }}" \
+          -var "logoImage:${{ inputs.branding-logo-image }}" \
+          -var "logoUrl:${{ inputs.branding-logo-url }}" \
+          -var "infracost_enabled:${{ fromJson(steps.atmos-settings.outputs.settings).enable-infracost }}" \
+          -var "driftModeEnabled:${{ inputs.drift-detection-mode-enabled }}" \
+          --output ${{ steps.vars.outputs.summary_file }} \
+          --log-level $([[ "${{ inputs.debug }}" == "true" ]] && echo "DEBUG" || echo "INFO") \
+          plan \
+          -patch \
+          -- \
+            bash -c "cat ${TERRAFORM_OUTPUT_FILE}"
+        fi
```

Comment:
Why do we need to run `tfcmt` twice?

Reply:
https://github.com/suzuki-shunsuke/tfcmt/issues/1412 this is so if the user wants a pr-comment you can enable it , there is no option on tfcmt to do it all at once